### DISCUSSION TITLE
Table 5, Panel 1

### DISCUSSION THREAD

**Discussion by bsmVRL** [cssi-pi-community/2022-meeting#21] [Panel 1 roundtable (Monday morning)]
panel 1 roundtable

**Comment by bsmVRL** [comment]
The NSF peer review also does not recognize robustifying/maintenance of software -- it is not considered "innovative". How does NSF plan to address this?

**Comment by DinoBektesevic** [comment]
I agree with the committee's analysis that lack of clearly defined career paths is one of the tenants at the core of this issue and that a good reward system can help.  

I'm afraid, however, that a what we will initially see is that CI positions are created using grants from programmes like CSSI, which disappear with the grant. While athis is a move towards positive solutions, it's not a long term solution to retention of professionals which will continue to be delayed because of institutional inertia and I have not heard good recommendations on how to change this in the future. 

Additionally, does NSF have any plans on tracking progress , such as longitudinal analysis of career paths of students/professionals in this field, or do we have good indicators of the scale (or the changes proposed today ha e) on this issue?

**Comment by gkthiruvathukal** [comment]
Our table decided to focus on one of the two strands: sustainability. We didn't think we could do justice to to both sustainability and diversity.

The following ideas were discussed:

- Addressing that universities have a stable source of revenue: students. But there is little stability when it comes to funding other contract positions: adjuncts/lecturers and RSEs.
- A possibility is to look at libraries/library science (librarians) as a model. But libraries are tied to the core mission (teaching) but provide a key role in research support.
- Question: Will TT faculty step up to recognize the importance of RSEs and recognizing their contributions? Among other things, making sure their contributions are fully counted as part of their annual evaluations, etc.
- NSF should do a proper study to understand the importance of RSEs and how to achieve sustainability not just in community outputs but the people behind them.

Please feel free to follow up with additions.
